src/core/internal/gc/impl/conservative/gc.d

```diff
@@ -2781,7 +2822,7 @@ Lmark:
             prepTime += (stop - start);
             start = stop;
 
-            if (doFork && !isFinal) // don't start a new fork during termination
+            if (doFork && !isFinal && !block) // don't start a new fork during termination
```

Comment:
Is this related to this commit (use clone() instead of fork()) or a different bug fix or similar?

Reply:
It's unrelated. Just the observation that it doesn't help to fork another process just to wait for it to return. We can just do it's work directly.